src/js/components/FileInput/README.md

```diff
@@ -55,12 +55,16 @@ Custom messages for FileInput. Used for accessibility by screen
 Whether to allow multiple files. If an object is supplied,
         'aggregageThreshold' indicates the maximum number of individual
         files to show. Above this, only a single message describing the
-        number of files will be shown.
+        number of files will be shown. If an object is supplied, 
+        'max' indicates the maximum number of files that can be added.
+        If the user has reached the max, no additional files will 
+        be able to be added.
```

Comment:
I'm wondering if we can prevent adding new files by disabling the "browse" button when the max is reached? I'm also wondering if we should then add a message indicating that the maximum number of files has been reached?